app/views/transferSummary.scala.html

```diff
@@ -53,7 +53,7 @@ <h1 class="govuk-heading-xl">@Messages("transferSummary.title")</h1>
             (Symbol("novalidate"), "")
             ) {
                 @CSRF.formField
-                <fieldset class="govuk-fieldset" aria-describedby="transfer-summary">
+                <fieldset class="govuk-fieldset" aria-label="final transfer confirmation">
```

Comment:
This will just tell the user (actually read it out to them) that the section at the bottom is the "final transfer confirmation" section, just so that there is a clear delineation between the summary and the confirmation.

Reply:
This is an interesting point. If we feel we have to make this clearer to users using assistive tech, then perhaps it's not clear enough to users who are reading the page?

Let's go with your idea for now, and raise it with the rest of the team in case they have thoughts on adding extra text to this page.

Reply:
I've just asked on Slack: https://tna-digital.slack.com/archives/CJYJYG54L/p1614592866127200